scripts/secrets/pull.sh

```
@@ -13,9 +13,50 @@ pull() {
+  # Fetch the secrets from the original path
+  local original_json=$EMPTY_JSON
+  local fetched_original_json
+  set +e
+  fetched_original_json=$(bao kv get -format=json "$bao_path" 2>/dev/null)
+  set -e
+  if [ -n "$fetched_original_json" ]; then
+    original_json=$fetched_original_json
+  fi
+
+  # If no secrets found, exit with an error
+  if [ "$generated_json" = $EMPTY_JSON ] && [ "$original_json" = $EMPTY_JSON ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)